frontend/env.d.ts

```diff
@@ -1 +1,3 @@
 /// <reference types="vite/client" />
+VITE_DEFAULT_LOCALE = en;
+VITE_FALLBACK_LOCALE = de;
```

Comment:
Just curious: Shouldn't the fallback be English as well? Most apps fallback to english...